specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -0,0 +1,247 @@
+# AppLifecycle - State Notifications
+
+Reunion introduces a new AppLifecycle component that provides a core set of functionality related to app activation and lifecycle. Much of this functionality is directly equivalent - or very similar - to existing functionality within the platform. However, there is value in abstracting this functionality so that all app types can use it, delivering it out-of-band from the OS, and making it available as open-source. The totality of AppLifecycle features in v1 is as follows (additional features may be added in later releases):
+-	Rich activation objects
+-	Selective single/multi-instancing
+-	System state and power notifications.
+
+This spec addresses the System State and Power Notifications APIs in the component.
+
+<br>
+
+## Background
+There are several existing APIs which apps can use to detect changes in system state, in order to participate in improved power usage and battery life. These APIs cover most  if not all of the battery/power state change scenarios that apps would care about - but there's no single API that is easily consumable by all app types. There's also no single API that's undocked from the OS. 
+
+This spec describes new APIs that address these issues by incorporating a comprehensive set of the [Windows.System.Power.PowerManager](https://docs.microsoft.com/en-us/uwp/api/Windows.System.Power.PowerManager) WinRT API in the Reunion AppLifecycle component, and include additional low-level notifications based on the Win32 [PowerSettingRegisterNotification](https://docs.microsoft.com/en-us/windows/win32/api/powersetting/nf-powersetting-powersettingregisternotification) and matching [PowerSettingUnregisterNotification](https://docs.microsoft.com/en-us/windows/win32/api/powersetting/nf-powersetting-powersettingunregisternotification) APIs.
+
+<br>
+
+## Description
```

Comment:
Somewhere in here we need a pattern saying things like "if you were using the Windows.*.PowerManager API before, you can switch to using Project Reunion's PowerManager by {x}"